source/components/tooltip/HxTooltip.js

```diff
@@ -0,0 +1,174 @@
+import { getPosition } from '../../lib/position';
```

Comment:
Based on changes in #104, you'll want to import `getPositionWithArrow` instead. Please rebase and update.